<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>790</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>720</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1031</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1031</width>
      <height>720</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255)
</string>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="page_5">
     <property name="minimumSize">
      <size>
       <width>1031</width>
       <height>720</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1031</width>
       <height>720</height>
      </size>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1301</width>
        <height>761</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>pics/background/Display-03.jpg</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="continue_button_2">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>510</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DIN 2014</family>
        <pointsize>33</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: white;
font: 33pt &quot;DIN 2014&quot;;</string>
      </property>
      <property name="text">
       <string>continue</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <property name="minimumSize">
      <size>
       <width>1031</width>
       <height>720</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1031</width>
       <height>720</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(&quot;background/Display-01.jpg&quot;);</string>
     </property>
     <widget class="QPushButton" name="continue_button">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>510</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DIN 2014</family>
        <pointsize>33</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: white;
font: 33pt &quot;DIN 2014&quot;;</string>
      </property>
      <property name="text">
       <string>continue</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1041</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>pics/background/Display-01.jpg</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="BackButton_7">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>101</y>
        <width>113</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 72, 24);
color: white;
font: 33pt &quot;DIN 2014&quot;;</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>continue_button</zorder>
     <zorder>BackButton_7</zorder>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1041</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>pics/background/Display-02.jpg</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="BackButton_8">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>101</y>
        <width>113</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 72, 24);
color: white;
font: 33pt &quot;DIN 2014&quot;;</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="continue_button_3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>510</y>
        <width>231</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DIN 2014</family>
        <pointsize>33</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: white;
font: 33pt &quot;DIN 2014&quot;;</string>
      </property>
      <property name="text">
       <string>continue</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="minimumSize">
      <size>
       <width>1031</width>
       <height>720</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1031</width>
       <height>720</height>
      </size>
     </property>
     <widget class="QPushButton" name="coachButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>360</y>
        <width>231</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DIN 2014</family>
        <pointsize>33</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 103, 2);
color: white;
font: 33pt &quot;DIN 2014&quot;;</string>
      </property>
      <property name="text">
       <string>COACH</string>
      </property>
     </widget>
     <widget class="QPushButton" name="userButton">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>360</y>
        <width>231</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>DIN 2014</family>
        <pointsize>33</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 103, 2);
color: white;
font: 33pt &quot;DIN 2014&quot;;</string>
      </property>
      <property name="text">
       <string>USER</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BackButton_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>101</y>
        <width>113</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 72, 24);
color: white;
font: 33pt &quot;DIN 2014&quot;;</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1041</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>pics/background/Display-06.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_7</zorder>
     <zorder>coachButton</zorder>
     <zorder>userButton</zorder>
     <zorder>BackButton_2</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <property name="minimumSize">
      <size>
       <width>1031</width>
       <height>720</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1031</width>
       <height>720</height>
      </size>
     </property>
     <widget class="QPushButton" name="BackButton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>101</y>
        <width>113</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 72, 24);
color: white;
font: 33pt &quot;DIN 2014&quot;;</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>200</y>
        <width>631</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QPushButton" name="stopRecording">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 103, 2);
color: white;
font: 33pt &quot;DIN 2014&quot;;</string>
         </property>
         <property name="text">
          <string>STOP RECORD</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="recordButton">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 103, 2);
color: white;
font: 33pt &quot;DIN 2014&quot;;</string>
         </property>
         <property name="text">
          <string>RECORD</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QListWidget" name="listWidget">
         <property name="styleSheet">
          <string notr="true">background-color: black;
color: white;
font: 15pt &quot;DIN 2014&quot;;</string>
         </property>
         <item>
          <property name="text">
           <string>Recording 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Recording 2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1032</width>
        <height>720</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>pics/background/Display-04.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_8</zorder>
     <zorder>BackButton</zorder>
     <zorder>gridLayoutWidget</zorder>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="BackButton_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>101</y>
        <width>113</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(229, 72, 24);
color: white;
font: 33pt &quot;DIN 2014&quot;;</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="runUser">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>520</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 103, 2);
color: white;
font: 33pt &quot;DIN 2014&quot;;</string>
      </property>
      <property name="text">
       <string>RUN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1041</width>
        <height>721</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>pics/background/Display-05.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_9</zorder>
     <zorder>BackButton_3</zorder>
     <zorder>runUser</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Recording">
   <property name="text">
    <string>New Recording</string>
   </property>
  </action>
  <action name="actionEnd_Recording">
   <property name="text">
    <string>End Recording</string>
   </property>
  </action>
  <action name="actionCoach">
   <property name="text">
    <string>Coach</string>
   </property>
  </action>
  <action name="actionCoach_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Coach</string>
   </property>
  </action>
  <action name="actionUser">
   <property name="text">
    <string>User</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
